lms/static/scripts/frontend_apps/components/dashboard/OrganizationActivity.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  Card,
+  CardContent,
+  CardHeader,
+  Link,
+} from '@hypothesis/frontend-shared';
+import { Link as RouterLink } from 'wouter-preact';
+
+import type { Courses } from '../../api-types';
+import { useConfig } from '../../config';
+import { urlPath, useAPIFetch } from '../../utils/api';
+import { replaceURLParams } from '../../utils/url';
+import OrderableActivityTable from './OrderableActivityTable';
+
+export type OrganizationActivityProps = {
+  organizationPublicId: string;
```

Comment:
Do we ever receive anything other than the "public" ID for an organization in the frontend? If not, we could use the term `organizationId` and make it always refer to the "public" ID.

Reply:
Yep, I thought about that, but since the terminology is going to change to `organization_public_id` in the backend to make sure it's not confused with the primary-key id, I thought it would be more clear to use the same terminology in the FE.

I don't have a strong opinion on this though. I think it will be clear enough regardless.